.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,29 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]    # [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.9]  # [2.7, 3.5, 3.6, 3.7, 3.8, pypy3]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: DigiPie/mongo-action@v1
+      - run: pip install black codespell flake8 isort pytest
+      - run: black --check . || true
+      # - run: black --diff . || true
+      # - if: matrix.python-version >= 3.6
+      #  run: |
+      #    pip install black
+      #    black --check . || true
+      - run: codespell --ignore-words-list="hist,optionn" --quiet-level=2 || true  # --skip=""
```

Comment:
maybe add a comment here to explain what the quiet-level is for? my understand is that the `|| true` is disabling this. Not sure I see the point on adding the line if it's always passing. How noisy is this at the moment?